src/main/java/com/github/manevolent/ts3j/protocol/socket/client/LocalTeamspeakClientSocket.java

```diff
@@ -433,9 +433,13 @@ public void process(AbstractTeamspeakClientSocket client,
         if (processor != null) {
             processor.process(client, singleCommand);
         } else if (singleCommand.getName().startsWith("notify")) {
-            TS3Event event = TS3Event.createEvent(singleCommand);
+            try {
+                TS3Event event = TS3Event.createEvent(singleCommand);
+                commandExecutionService.submit(() -> listeners.forEach(event::fire));
+            } catch (IllegalArgumentException exception) {
+                exception.printStackTrace();
```

Comment:
Can we propagate this exception to an exception listener?  I.e.,

`getExceptionHandler().accept(exception);`

Granted, I don't do this everywhere right now, but these especially seem like we should propagate them forward and notify the user if possible.

Reply:
do you mean something like an event for and unknow teamspeak event?

Reply:
That would also be a great solution :)

Reply:
done